astropy/time/tests/test_methods.py

```diff
@@ -479,9 +481,17 @@ def setup_class(cls):
             'masked': mjd + frac_masked
             }
 
+        cls.t2 = {
+            'not_masked': Time(mjd + frac, format='mjd', scale='utc',
```

Comment:
I'd prefer to do `Time(mjd, frac, format=...)` just not to loose precision up front.

Reply:
I copied this from `cls.t1` and `cls.jd` above, do you think I should modify those so that they also have full precision?

Reply:
Maybe just let this be if it was there too.